pkg/securelaunch/policy/policy.go

```diff
@@ -205,6 +202,12 @@ func parse(pf []byte) (*Policy, error) {
 	return p, nil
 }
 
+func measure(tpmHandle io.ReadWriteCloser, b []byte) error {
+	eventDesc := []byte(fmt.Sprintf("measured securelaunch policy file"))
```

Comment:
no need for fmt.Sprintf, just `[]byte("..")`

Reply:
Very Cool.